### DISCUSSION TITLE
Error message is not complete

### DISCUSSION THREAD

**Discussion by akrsnr** [dbeaver/dbeaver#13327] [Q&A] [answered]
Dbeaver doesn't show complete message as in ssms. Why ? Is there any settings for it or is it bug? It mustn't be specialized to ssms. 

![1](https://user-images.githubusercontent.com/17342831/127113933-60b8ed11-aaec-4bb4-b052-8706cb21b132.jpg)
![2](https://user-images.githubusercontent.com/17342831/127113937-83b83045-d732-4b4d-b742-57b7a8ae27b6.png)

#### System information: 
- Operating system (distribution) and version win10
- DBeaver version 21.0.0

#### Connection specification:
- Database name and version sql server 2017
- Driver name sql server jdbc
- Do you use tunnels or proxies (SSH, SOCKS, etc)?  no

**Comment by ShadelessFox** [comment, marked as answer]
Hello @akrsnr,

I believe this extra bit of information is provided by DBMS itself. In your case, I guess, it's SSMS.

In our case, we retrieve error messages from the driver, and they may lack some crucial parts like line numbers and procedure names.
